### DISCUSSION TITLE
Custom domain Github pages site is down

### DISCUSSION THREAD

**Discussion by julianeverett** [community/community#127089] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi there, my custom domain github pages site (https://thewildchoir.com) has been timing out all day today. The DNS query responses still look fine. Anyone know if there is a problem at github currently? (I know the status page is reporting green) Thanks

**Comment by Lucasigh** [comment]
this is happening with alot of people on BT or plusnet isp

**Reply by LiteBrite82** [reply, marked as answer]
Hi @julianeverett . 

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

https://github.com/orgs/community/discussions/127077

The Community Discussions Team

**Comment by Paul-Rettey** [comment]
Good morning,

Is there an SLA against this, just to give context our company runs assets for a number HE partners and this is seriously affecting our users experience so it would be good to get a time estimate against resolution.
